tests/src/common/WskCli.java

```diff
@@ -353,7 +352,7 @@ public String createAction(int expectedCode, String name, String artifact, Strin
         }
 
         if (shared) {
-            cmd = Util.concat(cmd, new String[] { "--shared" });
+            cmd = Util.concat(cmd, new String[] { "--shared", "yes"});
```

Comment:
Would have to update the Go CLI in order to make that change. I will open an issue.